content/collections/guides_and_surveys/en/resource-center-source-content.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 2d993365-2310-4343-ab54-d864173975eb
+blueprint: guides_and_survey
+title: 'Source Content'
+this_article_will_help_you:
+  - 'Include source content into the Resource Center'
+landing: false
+updated_by: 15756874-6eaf-4a8c-8779-bd4081ba41b6
+updated_at: 1750961154
+---
+The source content you select should be either internal help (documentation) repositories or specific external content. Documentation repositories are typically more useful as they provide a large selection of information that's widely applicable to each page with which your users interact. 
+
+- The Resource Center requires that you add documentation repositories and videos separately. 
+- Each documentation source requires slightly different information to successfully add it to the Resource Center. Follow the onscreen prompts for required fields.
+- If you don't see your specific documentation repository as an option, select the [Website](/docs/guides-and-surveys/resource-center-website-scraper) option to sync to a public website.
+- You can also suggest Resource Center content integrations through a Survey that's sent to Amplitude.
+
+**To add documentation content to the Resource Center** 
```

Comment:
Can this and the "Add video" section be headings? It will help break up the article and provide more targeted search results